meta-resin-common/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bb

```diff
@@ -22,7 +22,7 @@ do_configure[noexec] = "1"
 
 do_compile() {
     mkdir -p kernel_modules_headers
-    if [ "${TRANSLATED_TARGET_ARCH}" = "x86-64" ] || [ "${TRANSLATED_TARGET_ARCH}" = "i686" ]; then
+    if [[ "${TRANSLATED_TARGET_ARCH}" =~ [ix].*86 ]]; then
```

Comment:
Sadly this won't work with all shells as it is a bashism. You can test it with dash for example.